### PR TITLE
Algos

### PR DESCRIPTION
**Pull request by ghuinker**
This adds the rudementary workflow for the functioning algorith. Also changes to naming conventions and cleans up some backend aspects + endpoints + other stuff.

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -3,84 +3,40 @@
...
 from app.forms import AlgoRequestForm, DatabaseChoiceForm
-from app.algos import example_algo, gabe
+from app.algos import run as run_algo
+
+from app.databases import get_database_attributes, get_databases
 
 from app.models import Node
 from collections import Counter
 
 app_config = apps.get_app_config('app')
```
Comment (k98kumar) [inline app/views.py]: We don't need this line anymore if all dynamic attribute filtering has been nixed.

File: app/static/main.js [hunk 2]
```diff
@@ -228,6 +229,7 @@ function process() {
       return response.json();
     }).then((json) => {
       console.log(json);
+      document.querySelector('#chart').src = `data:image/png;base64,${json.chart}`
       if (json.data) {
     	  createSummary(json.data);
       }
```
Comment (k98kumar) [inline app/static/main.js:235]: Comment these lines for now (or just delete them). We're not using them currently and I don't want them to cause an issue in the future. Same for the next line.

File: app/views.py [hunk 3]
```diff
@@ -3,84 +3,40 @@
...
+
+def process(request):
+    """Return the algorithm function"""
+    # TODO: validate incoming data
+    # TODO: Use database map to get attributes
+    (chart, data) = run_algo(
+        method='kmeans',
+        time_frame=1,
+        proximity=[{'mass_1': 10, 'lumin_1': 20, 'rad_1': 20}],
+        diversity=[])
```
Comment (k98kumar) [inline app/views.py:35]: Should these be changed to not be hardcoded?

File: app/static/main.js (hunk 2 above)
Comment (ghuinker) [reply app/static/main.js:235]: This is part of your code so this feels like something you should delete or comment out.

File: app/views.py (hunk 3 above)
Comment (ghuinker) [reply app/views.py:35]: Eventually yes, as the commit implies I want to get an initial workflow going.

File: app/algos/__init__.py [hunk 4]
```diff
@@ -1,47 +1,24 @@
...
+from .kmeans import Kmeans
+from .dummy import DummyAlgo
+from app.matplot import plot_to_uri
 
-from kneed import KneeLocator
-from sklearn.datasets import make_blobs
-from sklearn.cluster import KMeans
-from sklearn.metrics import silhouette_score
-from sklearn.preprocessing import StandardScaler
+def run(method='kmeans', time_frame=1, proximity=[], diversity=[]):
```
Comment (jacobSpooner) [inline app/algos/__init__.py:5]: Does this need to be a dict of {name: val} for the attributes for proximity/diversity arrays? or do you just intend to receive the values knowing where each one is related?

Comment (ghuinker) [reply app/algos/__init__.py:5]: Haven't decided on that yet, but was going to work on that when I get the workflow functioning. I.e. next pull request.

Comment (jacobSpooner) [reply app/algos/__init__.py:5]: Thats fine, either way works on my end so do whatever is easiest for you

**Review by jacobSpooner — APPROVED** [review]
Looks fine on my end, I don't see anything that could cause issues down the line.

**Review by k98kumar — APPROVED** [review]

**Pull request merged**